tensorflow/python/ops/image_ops_test.py

```diff
@@ -4956,6 +4957,29 @@ def nms_func(box, score, iou_thres, score_thres):
       scores = constant_op.constant([0.9])
       nms_func(boxes, scores, iou_thres, [[score_thres]])
 
+  @test_util.xla_allow_fallback(
+      "non_max_suppression with dynamic output shape unsupported.")
+  def testTensors(self):
+    with context.eager_mode():
+      boxes_f = open("tensorflow/core/lib/tensors/boxes.tensor", "rb").read()
```

Comment:
This is difficult to read and use. Could you explicitly construct a tensor with a list (or with a numpy random value with fixed seed)?

Reply:
> This is difficult to read and use. Could you explicitly construct a tensor with a list (or with a numpy random value with fixed seed)?

Could I remove the test case if it's not necessary?

Reply:
> This is difficult to read and use. Could you explicitly construct a tensor with a list (or with a numpy random value with fixed seed)?

Done.